tests/api/users.test.js

```diff
@@ -230,5 +235,34 @@ describe("Users API endpoint", () => {
         expect.objectContaining({ error: "User not found" })
       );
     });
+
+    it("Should log admin activity if API call completed successfully", async () => {
```

Comment:
I suspect this test can be wrapped up by including `expect(logAdminRoleUpdate).toHaveBeenCalled();` in one of the previous tests

Reply:
There was none so I will rename to `Should successfully handle PUT request`